cloud/scope/managedcontrolplane.go

```diff
@@ -211,3 +212,19 @@ func (s *ManagedControlPlaneScope) IsIPv6Enabled() bool {
 func (s *ManagedControlPlaneScope) IsVnetManaged() bool {
 	return true
 }
+
+// APIServerLBName returns the API Server LB name.
+func (s *ManagedControlPlaneScope) APIServerLBName() string {
+	return ""
+}
+
+// IsAPIServerPrivate returns true if the API Server LB is of type Internal.
+// Currently always false as managed control planes do not currently implement private clusters.
+func (s *ManagedControlPlaneScope) IsAPIServerPrivate() bool {
+	 return false
+}
+
+
+func (s *ManagedControlPlaneScope) OutboundLBName(string) string {
+	return ""
```

Comment:
nit: actually `"kubernetes"`

but shouldn't be provisioned by CAPZ, only used for machines (not relevant to this pr)